### DISCUSSION TITLE
How do I stop seeing feedback from unwanted other people?

### DISCUSSION THREAD

**Discussion by leoheck** [community/community#15698] [Other Feature Feedback, Questions, & Ideas]
I just want to stop following some people. I want to build a clean and organized environment for my own sanity. 

I don't want to block people forever or purge them from planet earth. 

So, with that idea in mind, how do I just block the feedback from people that I don't want/know/talk?

These are the only possible choices. 
- Block user is not an option sometimes, for instance, if they are in the same organization as I.
- It is an abuse of my space, but the other people are not doing something wrong. The culprit is GitHub here.

![image](https://user-images.githubusercontent.com/1277920/165820748-7ac4c029-6c62-4683-b7bc-37b7eefb3878.png)

The [blocking users](https://docs.github.com/en/communities/maintaining-your-safety-on-github/blocking-a-user-from-your-personal-account) shows this
![image](https://user-images.githubusercontent.com/1277920/165629373-188fe19c-b972-40f6-bc31-735788fc13fb.png)

I am not looking for a way to block the user. I am looking for a way for GitHub not to flood my land page with useless information, and also with the information from people that I don't even know/want.

**Comment by Jxpro** [comment]
I have the same problem with you, and I am looking for a solution to the problem too.

**Comment by leoheck** [comment]
This is my suggestion. 
I don't hate people to block and kill them. I super valorize my own environment and peace of mind.

![image](https://user-images.githubusercontent.com/1277920/165820939-627c8168-c029-4a13-b963-c742e05a12d6.png)
